apps/src/code-studio/components/progress/SectionSelector.jsx

```diff
@@ -52,6 +52,7 @@ const SectionSelector = React.createClass({
     const { requireSelection, sections, selectedSectionId } = this.props;
     return (
       <select
+        className="uitest-sectionselect"
```

Comment:
Not very important, but why have `uitest` in the classname? Why not just `sectionsselect`?